### PR TITLE
tests/littlefs: Move from unittests to regular test

### PR DESCRIPTION
**Pull request by bergzand**
### Contribution description

another one related to #10187

### Testing procedure

Verify that `tests/pkg_littlefs` works: `BOARD=<your-favourite> make -C tests/pkg_littlefs flash test`

### Issues/PRs references

#10187

### CODE REVIEW

**Comment by miri64** [comment]
According to Murdock, some boards are to tiny for this tests and I believe the msp430 platforms need to be blacklisted (at least for `z1` I can't compile).

**Comment by miri64** [comment]
(you may squash immediately)

File: tests/pkg_littlefs/tests/01-run.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import os
+import sys
+from testrunner import run
+
+
+def testfunc(child):
+    board = os.environ['BOARD']
+    # Increase timeout on "real" hardware
+    timeout = 120 if board is not 'native' else -1
+    child.expect('OK \(\d+ tests\)', timeout=timeout)
```
Comment (cladmi) [inline tests/pkg_littlefs/tests/01-run.py]: Please also directly inline changing this regex to a raw string.
```suggestion
    child.expect(r'OK \(\d+ tests\)', timeout=timeout)
```

**Comment by bergzand** [comment]
Rebased!

**Comment by cladmi** [comment]
I could run the test on `arduino-mega2560 frdm-k64f frdm-kw41z msba2 nrf52dk nucleo-f103rb pba-d-01-kw2x sltb001a stm32f3discovery` but currently failed on the `mulle`.

However `mulle` was not passing `unittests` in master either.

https://ci-ilab.imp.fu-berlin.de/job/experimental-pull-request-tests/21/consoleText

I am running the test on the `IoT-LAB` nodes too.

**Comment by cladmi** [comment]
I could successfully run the test on `arduino-zero iotlab-m3 samr21-xpro b-l475e-iot01a microbit` on IoT-LAB.

Please squash before murdock start running it :)

**Comment by bergzand** [comment]
> Please squash before murdock start running it :)

squashed!

File: tests/pkg_littlefs/tests/01-run.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
```
Comment (cladmi) [inline tests/pkg_littlefs/tests/01-run.py]: Flake8 reports that this one is unused, you can squash it directly.
https://travis-ci.org/RIOT-OS/RIOT/builds/538272513#L441

Comment (bergzand) [reply tests/pkg_littlefs/tests/01-run.py]: Ah, should have noticed the warning locally. Fixed now

**Comment by cladmi** [comment]
Now we just wait for CI :)

**Comment by cladmi** [comment]
Same here, no problem for inline squashing the `BOARD_INSUFFICIENT_MEMORY`

**Comment by cladmi** [comment]
All green GO !

**Comment by bergzand** [comment]
Thanks!

**Pull request merged**